### PR TITLE
Edits to PGD4K PR5541

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

**Comment by ebgitelman** [comment]
@gvasquezvargas I took a fairly heavy hand with the Assembling content, as it was hard to follow and I felt could be both simplified and have more detail. If I haven't gotten your intentions right, let's go over it together.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx [hunk 1]
```diff
@@ -46,10 +46,10 @@ The example command:
 * Pulls images from the `k8s_enterprise_pgd` repository. 
 * Uses EBD Postgres Advanced Server 15.6.2 as the Postgres option. 
 * Uses PGD 5.4.1 as the Postgres Distributed version.
-* Uses 5.4.0 as the PGD proxy version.
+* Uses 5.4.0 as the PGD Proxy version.
 
 !!! Note Token 
-    Replace the token placeholder before running the command.
+    Before running the command, replace the token placeholder.
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx:52]: Order of operations.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx [hunk 2]
```diff
@@ -1,56 +1,52 @@
 ---
-title: 'Identifying your image names'
+title: 'Identifying image names'
 ---
 
-For a quick installation with the aim of testing the product, see the [Quickstart](/postgres_distributed_for_kubernetes/latest/quickstart/). 
+For a quick installation with the aim of testing the product, see the [Quick start](/postgres_distributed_for_kubernetes/latest/quickstart/).
 For more targeted testing or production purposes, 
-the following section provides an overview of how to select a specific operand and proxy image version that goes in line with your Postgres distribution.
+this overview describes how to select a specific operand and proxy image version that's appropriate for your Postgres distribution.
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx:7]: I *think* that's what you meant by "goes in line with"?

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx [hunk 3]
```diff
@@ -1,56 +1,52 @@
...
 
 ## Operand image name 
 
 See [Operand images](/postgres_distributed_for_kubernetes/latest/private_registries/#operand-images) for general information about the images.  
 
 ### Name elements 
 
-The operand image name is composed of three elements: the postgres type, EDB Postgres Distributed, and the build version.
+The operand image name is composed of the Postgres type, EDB Postgres Distributed, their version numbers, and the build version.
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx:15]: I think leaving the version numbers of the list was confusing.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx [hunk 4]
```diff
@@ -1,56 +1,52 @@
...
+**Postgres type:** The first part of the image name is the Postgres type you're using. The table shows how Postgres types correspond to this part of the image name.
 
 | Postgres type                | Image name            |
 |------------------------------|-----------------------|
 | EDB Postgres Advanced Server | edb-postgres-advanced |
 | EDB Postgres Extended Server | edb-postgres-extended |
 | PostgreSQL                   | postgresql            |
 
-**EDB Postgres Distributed:** The next step is identifying how Postgres Distributed translates to an image name. This is the same for all environment configurations: 
+**EDB Postgres Distributed:** For all environment configurations, `pgd` is the next part of the image name.
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx:25]: I think this is what you were saying? Avoid a one-row table.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx [hunk 5]
```diff
@@ -1,56 +1,52 @@
...
 
-| Product name             | Image name |
-|--------------------------|------------|
-| EDB Postgres Distributed | pgd        |
+**Version numbers:** The versions of the Postgres type and PGD, separated by a dash, are next in the image name.
 
-**Build version:** Last, the build version is 1, unless a new image is created in the scope of a security patch.
+**Build version:** The build version is 1, unless a new image is created in the scope of a security patch.
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx:29]: What if it *is* created in the scope of the security patch?

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx [hunk 6]
```diff
@@ -1,56 +1,52 @@
...
 |----------------------------|----------------------------------|-------|------------------------------------------|
 | EDB Postgres Advanced 16.2 | PGD 5.4.1                        | 1     | edb-postgres-advanced-pgd:16.2.0-5.4.1-1 |
 | EDB Postgres Extended 15.6 | PGD 5.4.0                        | 1     | edb-postgres-extended-pgd:15.6-5.4.0-1   |
 | PostgreSQL 16.2            | PGD 5.4.1                        | 1     | postgresql-pgd:16.2-5.4.1-1              |
 
-For an overview of Postgres compatibility, see [Platform compatibility](https://www.enterprisedb.com/resources/platform-compatibility).
-
 !!! Note Postgres version format
-    For PostgreSQL and EDB Extended Server images, the Postgres version is provided in `x.y` format, for example, 15.6. For EDB Postgres Advanced Server, the Postgres version is provided in `x.y.z` format, for example, 15.6.2.
+    For PostgreSQL and EDB Extended Server images, the Postgres version is in `x.y` format, for example, 15.6. For EDB Postgres Advanced Server, the Postgres version is in `x.y.z` format, for example, 15.6.2.
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx:42]: Passive voice was confusing here (is provided in). I wasn't sure if this was a user action. But in re-reading the content, I think this is about selecting the image name, not creating one.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx [hunk 7]
```diff
@@ -1,56 +1,52 @@
...
+
+For an overview of Postgres compatibility, see [Platform compatibility](https://www.enterprisedb.com/resources/platform-compatibility).
 
 ## Proxy image name 
 
-The proxy image name and version derive from EDB Postgres Distributed. See the [EDB Postgres Distributed release notes](/pgd/latest/rel_notes/) for an overview of PGD versions and corresponding PGD proxy versions. For example: 
+The proxy image name and version derive from EDB Postgres Distributed. See the [EDB Postgres Distributed release notes](/pgd/latest/rel_notes/) for an overview of PGD versions and corresponding PGD proxy versions. The table shows an example.
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx:48]: Don't use colons to introduce a table.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx [hunk 8]
```diff
@@ -6,9 +6,8 @@ navigation:
 ---
 
 If you plan on deploying a specific version of Postgres Distributed or a specific Postgres distribution or version, you will need to select the appropriate images and image versions.
-Gather the following information before [installing EDB Postgres Distributed for Kubernetes](/postgres_distributed_for_kubernetes/latest/installation_upgrade):
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx:9]: "Gather information" doesn't properly introduce the list, since each of the items is a complete action to perform.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx [hunk 9]
```diff
@@ -6,9 +6,8 @@ navigation:
...
-
-1. Identify your repository name and retrieve your user token as explained in [EDB private image registries](/postgres_distributed_for_kubernetes/latest/private_registries/).
-1. [Identify the image names for your environment](/postgres_distributed_for_kubernetes/latest/identify_images/identify_image_name). 
-1. [Assemble your deployment command](/postgres_distributed_for_kubernetes/latest/identify_images/assemble_command).
+Before [installing EDB Postgres Distributed for Kubernetes](/postgres_distributed_for_kubernetes/latest/installation_upgrade):
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx:9]: I changed this to an unordered list because it doesn't look like order matters.

Comment (djw-m) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx:9]: You can't do the last step without having done the preceding two steps.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx (hunk 5 above)
Comment (gvasquezvargas) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx:29]: I wish I could be more clear here, but unfortunately there is no place the user can go to see a list of the existing builds. Given that this is the first version of the product, the team has not prioritized this at the moment, hence why I am being vague. 
Do you think this is a bit better? 

```suggestion
**Build version:** The build version is 1. If a new image is created to include a security patch, the build version is replaced by a new number in consecutive order, for example, 2.
```

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx (hunk 6 above)
Comment (gvasquezvargas) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx:42]: You are correct; this is more about finding a different image name if you don't want to install what is provided by default (for example, because you are using an older Postgres version). I think your suggestion works well here.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx (hunk 9 above)
Comment (gvasquezvargas) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx:9]: Hi you two. 
Yes, I would like to keep the ordered list here. Especially the last step, assembling your deployment command, can't be done without the first two. Also, although 1 and 2 could be interchanged, I believe from a logical standpoint, it is easier to follow if 1 is done before two.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx [hunk 10]
```diff
@@ -6,9 +6,8 @@ navigation:
...
 
+- Identify your repository name and retrieve your user token as explained in [EDB private image registries](/postgres_distributed_for_kubernetes/latest/private_registries/).
```
Comment (gvasquezvargas) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx:11]:
```suggestion
1.  Identify your repository name and retrieve your user token as explained in [EDB private image registries](/postgres_distributed_for_kubernetes/latest/private_registries/).
```

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx [hunk 11]
```diff
@@ -6,9 +6,8 @@ navigation:
...
+- [Identify the image names for your environment](/postgres_distributed_for_kubernetes/latest/identify_images/identify_image_name).
```
Comment (gvasquezvargas) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx:12]:
```suggestion
1.  [Identify the image names for your environment](/postgres_distributed_for_kubernetes/latest/identify_images/identify_image_name).
```

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx [hunk 12]
```diff
@@ -6,9 +6,8 @@ navigation:
...
+- [Assemble your deployment command](/postgres_distributed_for_kubernetes/latest/identify_images/assemble_command).
```
Comment (gvasquezvargas) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx:13]:
```suggestion
1.  [Assemble your deployment command](/postgres_distributed_for_kubernetes/latest/identify_images/assemble_command).
```

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx (hunk 5 above)
Comment (ebgitelman) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx:29]: What if you just said:

**Build version:** Currently the only build version is 1.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx [hunk 13]
```diff
@@ -6,9 +6,8 @@ navigation:
...
-Gather the following information before [installing EDB Postgres Distributed for Kubernetes](/postgres_distributed_for_kubernetes/latest/installation_upgrade):
-
-1. Identify your repository name and retrieve your user token as explained in [EDB private image registries](/postgres_distributed_for_kubernetes/latest/private_registries/).
-1. [Identify the image names for your environment](/postgres_distributed_for_kubernetes/latest/identify_images/identify_image_name). 
-1. [Assemble your deployment command](/postgres_distributed_for_kubernetes/latest/identify_images/assemble_command).
+Before [installing EDB Postgres Distributed for Kubernetes](/postgres_distributed_for_kubernetes/latest/installation_upgrade):
 
+1.  Identify your repository name and retrieve your user token as explained in [EDB private image registries](/postgres_distributed_for_kubernetes/latest/private_registries/).
+1.  [Identify the image names for your environment](/postgres_distributed_for_kubernetes/latest/identify_images/identify_image_name).
+1.  [Assemble your deployment command](/postgres_distributed_for_kubernetes/latest/identify_images/assemble_command).
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx:13]: @djw-m Should these be relative paths, or does it not matter?

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx (hunk 5 above)
Comment (ebgitelman) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx:29]: Looks good!

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx (hunk 13 above)
Comment (ebgitelman) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx:13]: @gvasquezvargas @djw-m This is the only issue still remaining on this pull request. I know when we are referring to another EDB doc set we use this nomenclature for the link. But within the same doc set, I thought we were using relative paths? Or is this okay? What's our best practice?

Comment (djw-m) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx:13]: Same doc set, same sections. and this is the index so you don't even need paths for most of them....

In sequence:

../private_registries
identify_image_name
assemble_command

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx [hunk 14]
```diff
@@ -6,9 +6,8 @@ navigation:
...
 
 If you plan on deploying a specific version of Postgres Distributed or a specific Postgres distribution or version, you will need to select the appropriate images and image versions.
-Gather the following information before [installing EDB Postgres Distributed for Kubernetes](/postgres_distributed_for_kubernetes/latest/installation_upgrade):
-
-1. Identify your repository name and retrieve your user token as explained in [EDB private image registries](/postgres_distributed_for_kubernetes/latest/private_registries/).
-1. [Identify the image names for your environment](/postgres_distributed_for_kubernetes/latest/identify_images/identify_image_name). 
-1. [Assemble your deployment command](/postgres_distributed_for_kubernetes/latest/identify_images/assemble_command).
+Before [installing EDB Postgres Distributed for Kubernetes](/postgres_distributed_for_kubernetes/latest/installation_upgrade):
 
+1.  Identify your repository name and retrieve your user token as explained in [EDB private image registries](/postgres_distributed_for_kubernetes/latest/private_registries/).
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx:11]:
```suggestion
1.  Identify your repository name and retrieve your user token as explained in [EDB private image registries](../private_registries).
```

**Comment by ebgitelman** [comment]
@gvasquezvargas This should be ready to merge.

**Review by gvasquezvargas — APPROVED** [review]
LGTM! @ebgitelman sorry it took me this long, forgot that we had not merged this yet.
